frontend/apps/filemanager/filemanagerbookinfo.lua

```diff
@@ -158,8 +173,8 @@ function BookInfo:show(file, book_props)
             lines_nb = _("N/A")
             words_nb = _("N/A")
         end
-        table.insert(kv_pairs, { _("Current page lines:"), lines_nb })
-        table.insert(kv_pairs, { _("Current page words:"), words_nb })
+        table.insert(kv_pairs, { _("Page lines:"), lines_nb })
```

Comment:
I don't like this change. I guess you made it so that these items get a shorter title so you get more room for the values on the right.
But the title is a lot less clear than "current page lines", which leads to @Commodore64user undestanding wrong what it is.

Moreover, I really don't like your first screenshot, with the small key titles, the values really too far on the left, little air between title & values, and lots of ugly air/space on the right.
I know it's all quite random, and non-English UI can get things totally different - but with our English UI, I find it a lot nicer with the longer "Current page words" acting as the stop for the values, always making a bit of room in the middle between k/v (and not preventing longer values to eat in that room and meet the key title).

Reply:
But 'Current page words' (same for the other) doesn't really mean anything, perhaps 'words in page' or 'words in current page'.

Reply:
Either words/lines in page or https://github.com/koreader/koreader/pull/12313#discussion_r1710106163 sounds fine to me ;).

Reply:
How about we save 1 line and call it `Page statistics:` in the left-hand column, and put "X lines; Y words." in the right-hand side?